uqcsbot/text.py

```diff
@@ -76,6 +77,31 @@ async def caesar(self, interaction: discord.Interaction, message: str, distance:
 
         await interaction.response.send_message(result)
 
+    @app_commands.command()
+    @app_commands.describe(message="Input string", encoding="Character encoding to use, defaults to UTF-8")
+    async def hexify(self, interaction: discord.Interaction, message: str, encoding: Optional[str] = "utf-8"):
```

Comment:
Similarly to above, encoding should be of type `Optional[Literal["utf-8", "ascii"]]` or similar.

Reply:
Same as above. Completed with autocomplete